faq.md

```diff
@@ -78,6 +81,35 @@ and more. Use cases include complex monitoring and analytics; predicting the per
 behavior of applications, models, consumers and connected machines; powering operational
 analytical workflows and dashboards; for QA and performance testing. [[Top]](#top)
 
+### **Why should I choose Timescale Cloud?** [](why-cloud)
+If you want all the benefits of TimescaleDB without the hassle of installing,
+maintaining, and administering the database itself, then Timescale Cloud is the
+best choice for you.
+
+Timescale Cloud manages all of the operational elements of your database so you can 
+focus on building your applications and not making sure the infrastructure works. We 
+ensure you have a secure, high availability environment where we manage the infrastructure 
+all the way down to setting up replications, point-in-time recovery, read replicas, 
+backups, and more. [[Top]](#top)
+
+### **Which cloud providers and regions are supported by Timescale Cloud?** [](which-cloud-regions)
+Timescale Cloud is available in the following Clouds and Regions:
+
+* **Amazon Web Services (AWS)**: N. Virginia (`us-east-1`), Ohio (`us-east-2`), 
+N. California (`us-west-1`), Oregon (`us-west-2`), São Paulo (`sa-east-1`), 
+Stockholm (`eu-north-1`), Ireland (`eu-west-1`), London (`eu-west-2`), Paris (`eu-west-3`), 
+Frankfurt (`eu-central-1`), Canada (`ca-central-1`), Singapore (`ap-southeast-1`), 
+Sydney (`ap-southeast-2`), Tokyo (`ap-northeast-1`), Mumbai (`ap-south-1`), Seoul (`ap-northeast-2`)
+* **Microsoft Azure (Azure)**: East US, East US 2, West Europe, Southeast Asia
+* **Google Cloud Platform (GCP)**: Northern Virginia, Los Angeles, South Carolina, 
+Iowa, Oregon, São Paulo, Zürich, London, Frankfurt, Finland, Belgium, Netherlands, 
+Montréal, Sydney, Taiwan, Mumbai, Hong Kong, Tokyo, Singapore
+[[Top]](#top)
+
+### **What is the difference between Timescale, TimescaleDB, and Timescale Cloud?** [](what-does-timescale-mean)
+**Timescale** is our company. We build a category-defining time-series database called **TimescaleDB**. 
```

Comment:
not sure "our" is the right choice here since this usage would most likely exclude the reader